templates/master/00-master/baremetal/files/NetworkManager-dns-vip-prepender.yaml

```diff
@@ -0,0 +1,44 @@
+filesystem: "root"
+mode: 0755
+path: "/etc/NetworkManager/dispatcher.d/pre-up.d/dns-vip-prepender"
+contents:
+  inline: |
+    #!/bin/bash
+    
+    IFACE=$1
+    STATUS=$2
+    
+    case "$STATUS" in
+        pre-up)
+        logger -s "NM dns-vip-prepender triggered by pre-upping ${1}."
+        API_VIP="{{ .Infra.Status.PlatformStatus.BareMetal.APIServerInternalIP }}"
```

Comment:
@bcrochet, it supposed to be DNS_VIP instead of API_VIP , right?

Reply:
Yes. I've fixed that. I'll manually merge this in.